source/_components/sensor.thermoworks_smoke.markdown

```diff
@@ -0,0 +1,127 @@
+---
+layout: page
+title: "Thermoworks Smoke Sensor"
+description: "Pulls temperature data for a Thermoworks Smoke Thermometer connected with Smoke Gateway."
+date: 2018-08-22 17:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: home-assistant.png
```

Comment:
This is the logo for Home Assistant internal platforms/components. Should be the logo of Thermoworks if possible

Reply:
I added this logo `source/images/components/thermoworks/thermoworks-logo.jpg` how would I specify the path.

Reply:
disregard, i found supported brands. will convert to png and switch.